From d0d3e6f1a9ffbac91f24cc06abce6dcad6509a6b Mon Sep 17 00:00:00 2001
From: Julien Jerphanion <git@jjerphan.xyz>
Date: Tue, 6 Dec 2022 10:24:26 +0100
Subject: [PATCH 1/2] MAINT Include tuple in 'utils/strided_iters.hpp'

This include directive seems to have been forgotten and it is impossible
for me to compile dpctl without it.
---
 dpctl/tensor/libtensor/include/utils/strided_iters.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dpctl/tensor/libtensor/include/utils/strided_iters.hpp b/dpctl/tensor/libtensor/include/utils/strided_iters.hpp
index c24ed54941..595ad3f4a7 100644
--- a/dpctl/tensor/libtensor/include/utils/strided_iters.hpp
+++ b/dpctl/tensor/libtensor/include/utils/strided_iters.hpp
@@ -30,6 +30,7 @@
 #include <array>
 #include <numeric> // std::iota
 #include <vector>
+#include <tuple>
 
 /* An N-dimensional array can be stored in a single
  * contiguous chunk of memory by contiguously laying

From f75ccd8c8aeaae92f0cb03874f93be5e7a63a6a7 Mon Sep 17 00:00:00 2001
From: Julien Jerphanion <git@jjerphan.xyz>
Date: Tue, 6 Dec 2022 14:45:40 +0100
Subject: [PATCH 2/2] Make clang-format happy

---
 dpctl/tensor/libtensor/include/utils/strided_iters.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dpctl/tensor/libtensor/include/utils/strided_iters.hpp b/dpctl/tensor/libtensor/include/utils/strided_iters.hpp
index 595ad3f4a7..0abd7f4f2a 100644
--- a/dpctl/tensor/libtensor/include/utils/strided_iters.hpp
+++ b/dpctl/tensor/libtensor/include/utils/strided_iters.hpp
@@ -29,8 +29,8 @@
 #include <algorithm> // sort
 #include <array>
 #include <numeric> // std::iota
-#include <vector>
 #include <tuple>
+#include <vector>
 
 /* An N-dimensional array can be stored in a single
  * contiguous chunk of memory by contiguously laying
